estimation_tracker/tracker/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
```

Comment:
Could you add some unittests for the view and the methods you think are important to test?

Reply:
sure